sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/GetHashCodeMutableCodeFixProvider.cs

```diff
@@ -49,41 +52,65 @@ public sealed override FixAllProvider GetFixAllProvider()
         protected sealed override async Task RegisterCodeFixesAsync(SyntaxNode root, CodeFixContext context)
         {
             var diagnostic = context.Diagnostics.First();
-            var diagnosticSpan = diagnostic.Location.SourceSpan;
-            var identifierName = root.FindNode(diagnosticSpan, getInnermostNodeForTie: true) as IdentifierNameSyntax;
-            if (identifierName == null)
+
+            var identifiersToFix = diagnostic.AdditionalLocations
+                .Select(location => location.SourceSpan)
+                .Select(diagnosticSpan => root.FindNode(diagnosticSpan, getInnermostNodeForTie: true) as IdentifierNameSyntax)
+                .Where(idenfitierName => idenfitierName != null)
+                .ToList();
+
+            if (identifiersToFix.Count == 0)
             {
                 return;
             }
 
             var semanticModel = await context.Document.GetSemanticModelAsync(context.CancellationToken).ConfigureAwait(false);
+            var allFieldDeclarationTasks = identifiersToFix.Select(identifier => GetFieldDeclarationSyntax(semanticModel, identifier, context.CancellationToken));

```

Comment:
We should reduce our "line too long" rule limit to 130 chars. GitHub is showing scrollbar for longer lines...